siriuspy/siriuspy/csdevice/pwrsupply.py

```diff
@@ -194,8 +194,10 @@ class ETypes(_cutil.ETypes):
     HARD_INTLCK_FAC_2S_ACDC = (
         'Sobre-tensão no banco de capacitores',
         'Sobre-corrente na saída do retificador',
-        'Interlock da placa IIB', 'Reserved',
-        'Reserved', 'Reserved', 'Reserved', 'Reserved',
+        'Falha no contador de entrada AC trifásica'
```

Comment:
isn't it 'contator' instead of 'contador'?

Reply:
it is! thanks! 
fixed.